packages/jest-cli/src/plugins/update_snapshots.js

```diff
@@ -7,31 +7,35 @@
  * @flow
  */
 import type {GlobalConfig} from 'types/Config';
-import WatchPlugin from '../watch_plugin';
+import BaseWatchPlugin from '../base_watch_plugin';
 import type {JestHookSubscriber} from '../jest_hooks';
 
-class UpdateSnapshotsPlugin extends WatchPlugin {
+class UpdateSnapshotsPlugin extends BaseWatchPlugin {
   _hasSnapshotFailure: boolean;
-  showPrompt(
+  runInteractive(
     globalConfig: GlobalConfig,
     updateConfigAndRun: Function,
   ): Promise<boolean> {
     updateConfigAndRun({updateSnapshot: 'all'});
     return Promise.resolve(false);
   }
 
-  registerHooks(hooks: JestHookSubscriber) {
+  apply(hooks: JestHookSubscriber) {
+    this._hasSnapshotFailure = true;
```

Comment:
Is this necessary/intentional or a leftover from debugging?

Reply:
Nice catch